src/game/scripts/vscripts/ingame.lua

```diff
@@ -541,6 +572,31 @@ function Ingame:fixRuneBug()
     end, nil)
 end
 
+-- Called every 0.1 second to check and convert consumable items into actual consumable items
+function Ingame:CheckConsumableItems()
+    local itemTable = LoadKeyValues('scripts/kv/consumable_items.kv')
+    for i=0,PlayerResource:GetTeamPlayerCount() do
+        if PlayerResource:IsValidTeamPlayerID(i) and not util:isPlayerBot(i) then
+            local hero = PlayerResource:GetSelectedHeroEntity(i)
+            if hero and IsValidEntity(hero) then
+                for i=0,14 do
+                    local hItem = hero:GetItemInSlot(i)
+                    if hItem then
+                        local name = hItem:GetAbilityName()
+                        if itemTable[name] then
+                            hero:RemoveItem(hItem)
+                            hero:AddItemByName(name.."_consumable")
+                            local nSlot, hUseless = hero:FindItemByNameEverywhere(name.."_consumable")
```

Comment:
It's the consumable item, it's not used, but the function returns it anyways so I thought I'd keep it around for future problems.